datacontract/export/sql_type_converter.py

```diff
@@ -108,6 +110,44 @@ def convert_type_to_postgres(field: Field) -> None | str:
     return None
 
 
+# dataframe data types:
+# https://spark.apache.org/docs/latest/sql-ref-datatypes.html
+def convert_to_dataframe(field: Field) -> None | str:
+    type = field.type
```

Comment:
I think we should also support a config option `config.dataframeType`:

Could you add this?

    if field.config and "dataframeType" in field.config:
        return field.config["dataframeType"]